.github/workflows/codeowners.yml

```diff
@@ -0,0 +1,19 @@
+name: Ping maintainers
+
+on:
+  pull_request:
+    types: [opened, synchronize, ready_for_review]
+  push:
+    types: [opened, synchronize, ready_for_review]
+
+jobs:
+  codenotify:
+    runs-on: ubuntu-latest
+    name: codenotify
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: sourcegraph/codenotify@v0.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.CODENOTIFY_GITHUB_TOKEN }}
```

Comment:
Who is setting this token?

Reply:
I thought I had responded to this but GitHub does not show it anymore. Did you get anything?

Reply:
It needs to be set by an admin.

Reply:
> I thought I had responded to this but GitHub does not show it anymore. Did you get anything?

Ah, sorry I misread this. No I didn't see a response.

Reply:
Interesting. Anyways, I don't know what I'm doing there and why, I mostly copied some example configuration. If I understand it correctly, the token is required to be able to ping teams. So it should be able to ping individuals even without.